.circleci/config.yml

```diff
@@ -52,44 +32,27 @@ jobs:
           root: ~/repo
           <<: *whitelist
 
-  build_base_docker:
-    <<: *defaults
-
-    steps:
-      - attach_workspace:
-          at: ~/repo
-
-      - setup_remote_docker
-
-      - run:
-          name: Build Base Docker
-          # install dependencies for loading ecs task definitions
-          command: |
-            eval $(aws ecr get-login --no-include-email)
-            globality-build build-gen local
-            globality-build docker --repo python-library
-
   test:
-    <<: *defaults
+    working_directory: ~/repo
+    docker:
+      - image: circleci/python:3.7.0
+      - image: circleci/postgres:9.6.5-alpine-ram
+        environment:
+          POSTGRES_USER: charmander
+          POSTGRES_DB: charmander_test_db
 
     steps:
       - attach_workspace:
           at: ~/repo
 
-      - setup_remote_docker
-      - run:
-          name: pull and run database - postgres
-          command: |
-            docker pull postgres
-            docker run -d --name charmander_db -e POSTGRES_DB=charmander_test_db -e POSTGRES_USER=charmander postgres
       - run:
           name: Test code
           command: |
-            docker create -v /src/charmander/tests/ --name service_tests alpine:3.4 /bin/true
```

Comment:
Compare this diff with where this actually comes from in `globality-build`: https://github.com/globality-corp/globality-build/blob/master/globality_build/templates/build/python-service/entrypoint.sh#L39